diff -up mariadb-10.0.27/mysql-test/t/openssl_1.test.patch36 mariadb-10.0.27/mysql-test/t/openssl_1.test
diff -up mariadb-10.0.27/mysql-test/t/ssl_8k_key.test.patch36 mariadb-10.0.27/mysql-test/t/ssl_8k_key.test
--- mariadb-10.0.27/mysql-test/t/ssl_8k_key.test.patch36	2016-08-24 17:07:41.000000000 +0200
+++ mariadb-10.0.27/mysql-test/t/ssl_8k_key.test	2016-09-18 21:48:16.098687451 +0200
@@ -5,6 +5,7 @@
 #
 # Bug#29784 YaSSL assertion failure when reading 8k key.
 #
+--replace_result DHE-RSA-AES256-GCM-SHA384 DHE-RSA-AES256-SHA DHE-RSA-CHACHA20-POLY1305 DHE-RSA-AES256-SHA AES256-GCM-SHA384 DHE-RSA-AES256-SHA
 --exec $MYSQL --ssl --ssl-key=$MYSQL_TEST_DIR/std_data/client-key.pem --ssl-cert=$MYSQL_TEST_DIR/std_data/client-cert.pem -e "SHOW STATUS LIKE 'ssl_Cipher'" 2>&1
 
 ##  This test file is for testing encrypted communication only, not other
diff -up mariadb-10.0.27/mysql-test/t/ssl_ca.test.patch36 mariadb-10.0.27/mysql-test/t/ssl_ca.test
diff -up mariadb-10.0.27/mysql-test/t/ssl_compress.test.patch36 mariadb-10.0.27/mysql-test/t/ssl_compress.test
diff -up mariadb-10.0.27/mysql-test/t/ssl.test.patch36 mariadb-10.0.27/mysql-test/t/ssl.test
diff -up mariadb-10.0.27/mysql-test/t/ssl_timeout.test.patch36 mariadb-10.0.27/mysql-test/t/ssl_timeout.test
